src/polyswarmd/config.py

```diff
@@ -256,10 +257,11 @@ def from_config_file(cls, filename):
 
         commmunity = config.get('community')
         ipfs_uri = config.get('ipfs_uri')
+        artifact_limit = config.get('artifact_limit', 256)
```

Comment:
why default to 256 if you are changing stuff to limit it to 1? Maybe default to 1?

Reply:
The default should match the contract IMO. If we want less, we should explicitly set it.

Reply:
ok